lib/compress/zstd_compress.c

```diff
@@ -2097,7 +2064,7 @@ size_t ZSTD_compressContinue (ZSTD_CCtx* cctx,
 size_t ZSTD_getBlockSize(const ZSTD_CCtx* cctx)
 {
     ZSTD_compressionParameters const cParams =
-            ZSTD_getCParamsFromCCtxParams(cctx->appliedParams, 0, 0);
+            ZSTD_getCParamsFromCCtxParams(&(cctx->appliedParams), 0, 0);
```

Comment:
Why can't we just use `cctx->appliedParams.cParams` here? Shouldn't `appliedParams` already be correctly adjusted?

Reply:
I wasn't completely sure, but indeed, `cParams` are supposed already built at this stage.
I'll replace this extraction by an `assert()`.